lib/kubeclient/common.rb

```diff
@@ -42,14 +48,15 @@ module ClientMixin
     attr_reader :headers
     attr_reader :discovered
 
-    def initialize_client(
+    def initialize_client( # rubocop:disable Metrics/ParameterLists
```

Comment:
I guess we can live with that. We should make travis warn and not fail on rubocop at some point